### ISSUE TITLE
[v2 QUESTION]: Molecule-Level 2D Features

### ISSUE THREAD

**Issue by jia-huang** [chemprop/chemprop#1127]
In the following command, why does the validation loss always show the lowest value in the first epoch after I add the --molecule-featurizers rdkit_2d feature? The predicted activity probabilities for each molecule are generally below 0.3.

chemprop train --data-path ../chemprop_data/FDA/FDA-smiles.csv --task-type classification --output-dir FDA_has_200f_checkpoints --epochs 30 --molecule-featurizers rdkit_2d

![截图１](https://github.com/user-attachments/assets/8074bc93-e9cc-48ac-ac76-6f30379f9998)
![截图２](https://github.com/user-attachments/assets/32213d65-e450-4443-ae8d-c9d5c1dc2a4e)

**Comment by JacksonBurns** [comment]
It's possible that the model stops improving after the first epoch. If your dataset contains all or almost all negative labels, the model would never learn the positive label and this could cause the performance to stagnate after the first epoch.
